gateway/requests/prometheus_test.go

```diff
@@ -1,4 +1,16 @@
-{
+// Copyright (c) Alex Ellis 2017. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package requests
+
+import (
+	"encoding/json"
+	"testing"
+)
+
+// TestUnmarshallAlert is an exploratory test from TDD'ing the struct to parse a Prometheus alert
+func TestUnmarshallAlert(t *testing.T) {
+	file := []byte(`{
     "receiver": "scale-up",
```

Comment:
Is this the same as the original file 1-for-1?

Reply:
I thought I added a comment on this... sorry.
Originally, this was reading a file called `test_alert.json`. I put the content directly inside the test because having a `test_alert.json` inside the `requests` go package for test seemed strange.
Another way would be to leave the `test_alert.json` inside `tests` directory and read the file with relative path. Imo, caring about the relative path across packages seemed a bit fragile so I put the content inside the unit test.
Do you have suggestions? Thanks.

Reply:
It's fine as you have it - I just wanted to make sure the test data was representative. Sounds like it is